### PR TITLE
rg-40: Add Ability Edit Template

### PR DESCRIPTION
**Pull request by siefimov**
You can check UI on Preview component.
image

### CODE REVIEW

**Comment by Vitaliy-Kalenichenko** [comment]
Resolve conflicts

**Comment by siefimov** [comment]
Conflicts resolved.

**Review by Vitaliy-Kalenichenko — APPROVED** [review]

File: frontend/src/bundles/edit-temlate/types/types.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export {
+    type TemplateBlockSettings,
+    type TemplateDto,
+    type TemplateUpdateItemRequestDto,
+    type TemplateUpdateItemResponseDto,
+} from '../../../../../shared/build/bundles/templates/templates.js';
```
Comment (apofis96) [inline frontend/src/bundles/edit-temlate/types/types.ts:6]: Don't use relative path for shared

**Review by apofis96 — CHANGES_REQUESTED** [review]

**Comment by Vitaliy-Kalenichenko** [comment]
Conflicts :(

File: frontend/src/bundles/edit-temlate/types/types.ts (hunk 1 above)
Comment (siefimov) [reply frontend/src/bundles/edit-temlate/types/types.ts]: Fixed.

**Pull request merged**